Community/add_mac_address/templates/add_mac_address_page.html

```diff
@@ -0,0 +1,24 @@
+<!-- Copyright 2018 BlueCat Networks. All rights reserved. -->
```

Comment:
Please change date to 2019 as well as other instances in the other files (one instance of 2017).

Reply:
Dates changed to 2019